routes/project/location/postgres.rb

```diff
@@ -234,6 +234,28 @@ class Clover
         end
       end
 
+      r.post "set-maintenance-window" do
+        authorize("Postgres:edit", pg.id)
+
+        allowed_optional_parameters = ["maintenance_window_start_at"]
+        request_body_params = validate_request_params([], allowed_optional_parameters)
+
+        maintenance_window_start_at = request_body_params["maintenance_window_start_at"]
+        maintenance_window_start_at = nil if web? && maintenance_window_start_at == ""
+        maintenance_window_start_at = Validation.validate_postgres_maintenance_window_start_at(maintenance_window_start_at)
+
+        DB.transaction do
+          pg.update(maintenance_window_start_at:)
+        end
```

Comment:
I don't think this transaction is necessary. `Model#update` uses a transaction already, and it is the only call inside the block.